k8s/c2d_deployer.Makefile

```diff
@@ -0,0 +1,107 @@
+#
+# Used with app_v2 Makefile for deployer schema v2.
+#
+
+ifndef __C2D_DEPLOYER_MAKEFILE__
+
+__C2D_DEPLOYER_MAKEFILE__:= included
+
+##### Common variables #####
+
+APP_DEPLOYER_IMAGE ?= $(REGISTRY)/$(APP_ID)/deployer:$(RELEASE)
+APP_DEPLOYER_IMAGE_TRACK_TAG ?= $(REGISTRY)/$(APP_ID)/deployer:$(TRACK)
+TESTER_IMAGE ?= $(REGISTRY)/$(APP_ID)/tester:$(RELEASE)
+APP_GCS_PATH ?= $(GCS_URL)/$(APP_ID)/$(TRACK)
+
+
+include ../app_v2.Makefile
+
+
+$(info ---- TRACK = $(TRACK))
+$(info ---- RELEASE = $(RELEASE))
+$(info ---- APP IMAGE = $(image-$(APP_ID)))
+
+
+##### Helper targets #####
+
+app/build:: .build/$(APP_ID)/VERSION \
+            $(APP_ID) \
```

Comment:
I think it could be better to keep the old name. Please see my comment about `.PHONY` targets.